docs/index.rst

```diff
@@ -8,7 +8,10 @@ resources targeted for edX learners, educators, researchers, and Open edX operat
 .. toctree::
     :hidden:
 
+    Open edX Proposals <https://open-edx-proposals.readthedocs.io/en/latest/>
```

Comment:
@nasthagiri: See the screenshots in this PR to better understand the purpose of the redundancy in this hidden toc.